Subject: [PATCH] 10_0_fix_js
---
Index: src/main/webapp/resources/js/topjava.users.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/webapp/resources/js/topjava.users.js b/src/main/webapp/resources/js/topjava.users.js
--- a/src/main/webapp/resources/js/topjava.users.js	(revision fec18cb74894c6445c19a50a9a256e81e84f1a3a)
+++ b/src/main/webapp/resources/js/topjava.users.js	(date 1755182632042)
@@ -41,7 +41,7 @@
                     "data": "email",
                     "render": function (data, type, row) {
                         if (type === "display") {
-                            return "<a href='mailto:${data}'>${data}</a>";
+                            return `<a href="mailto:${data}">${data}</a>`;
                         }
                         return data;
                     }
@@ -53,7 +53,7 @@
                     "data": "enabled",
                     "render": function (data, type, row) {
                         if (type === "display") {
-                            return "<input type='checkbox' ${data ? 'checked' : ''} onclick='enable($(this),${row.id});'/>";
+                            return `<input type='checkbox' ${data ? 'checked' : ''} onclick='enable($(this),${row.id});'/>`;
                         }
                         return data;
                     }
Index: src/main/webapp/resources/js/topjava.common.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/webapp/resources/js/topjava.common.js b/src/main/webapp/resources/js/topjava.common.js
--- a/src/main/webapp/resources/js/topjava.common.js	(revision fec18cb74894c6445c19a50a9a256e81e84f1a3a)
+++ b/src/main/webapp/resources/js/topjava.common.js	(date 1755182632033)
@@ -23,7 +23,7 @@
     $("#modalTitle").html(i18n["editTitle"]);
     $.get(ctx.ajaxUrl + id, function (data) {
         $.each(data, function (key, value) {
-            form.find("input[name='${key}']").val(value);
+            form.find(`input[name='${key}']`).val(value);
         });
         $('#editRow').modal();
     });
@@ -69,7 +69,7 @@
 function successNoty(key) {
     closeNoty();
     new Noty({
-        text: "<span class='fa fa-lg fa-check'></span> &nbsp;${i18n[key]}",
+        text: `<span class='fa fa-lg fa-check'></span> &nbsp;${i18n[key]}`,
         type: 'success',
         layout: "bottomRight",
         timeout: 1000
@@ -78,20 +78,20 @@
 
 function renderEditBtn(data, type, row) {
     if (type === "display") {
-        return "<a onclick='updateRow(${row.id});'><span class='fa fa-pencil'></span></a>";
+        return `<a onclick='updateRow(${row.id});'><span class='fa fa-pencil'></span></a>`;
     }
 }
 
 function renderDeleteBtn(data, type, row) {
     if (type === "display") {
-        return "<a onclick='deleteRow(${row.id});'><span class='fa fa-remove'></span></a>";
+        return `<a onclick='deleteRow(${row.id});'><span class='fa fa-remove'></span></a>`;
     }
 }
 
 function failNoty(jqXHR) {
     closeNoty();
     failedNote = new Noty({
-        text: "<span class='fa fa-lg fa-exclamation-circle'></span> &nbsp;${i18n['common.errorStatus']}: ${jqXHR.status}${jqXHR.responseJSON ? '<br>' + jqXHR.responseJSON : ''}",
+        text: `<span class='fa fa-lg fa-exclamation-circle'></span> &nbsp;${i18n['common.errorStatus']}: ${jqXHR.status}${jqXHR.responseJSON ? '<br>' + jqXHR.responseJSON : ''}`,
         type: "error",
         layout: "bottomRight"
     });
